tests/commands/build.spec.ts

```diff
@@ -115,6 +116,85 @@ describe('balena build', function() {
 		});
 	});
 
+	it('should create the expected tar stream (--emulated)', async () => {
+		const projectPath = path.join(projectsPath, 'no-docker-compose', 'basic');
+		const transposedDockerfile =
+			stripIndent`
+			FROM busybox
+			COPY [".balena/qemu-execve","/tmp/qemu-execve"]
+			COPY ["./src","/usr/src/"]
+			RUN ["/tmp/qemu-execve","-execve","/bin/sh","-c","chmod a+x /usr/src/*.sh"]
+			CMD ["/usr/src/start.sh"]` + '\n';
+		const expectedFiles: ExpectedTarStreamFiles = {
+			'src/start.sh': { fileSize: 89, type: 'file' },
+			'src/windows-crlf.sh': { fileSize: 70, type: 'file' },
+			Dockerfile: {
+				fileSize: transposedDockerfile.length,
+				type: 'file',
+				contents: transposedDockerfile,
+			},
+			'Dockerfile-alt': { fileSize: 30, type: 'file' },
+		};
+		const responseFilename = 'build-POST.json';
+		const responseBody = await fs.readFile(
+			path.join(dockerResponsePath, responseFilename),
+			'utf8',
+		);
+		const expectedResponseLines = [
+			`[Info] No "docker-compose.yml" file found at "${projectPath}"`,
+			`[Info] Creating default composition with source: "${projectPath}"`,
+			'[Info] Building for rpi/raspberry-pi',
+			'[Info] Emulation is enabled',
+			'[Build] main Image size: 1.14 MB',
+			'[Success] Build succeeded!',
+		];
+		if (isWindows) {
+			expectedResponseLines.push(
+				`[Warn] CRLF (Windows) line endings detected in file: ${path.join(
+					projectPath,
+					'src',
+					'windows-crlf.sh',
+				)}`,
+				'[Warn] Windows-format line endings were detected in some files. Consider using the `--convert-eol` option.',
+			);
+		}
+		const arch = 'rpi';
+		const deviceType = 'raspberry-pi';
+		const fsModPath = 'mz/fs';
+		const fsMod = await import(fsModPath);
+		const qemuModPath = '../../build/utils/qemu';
+		const qemuMod = require(qemuModPath);
+		const qemuBinPath: string = await qemuMod.getQemuPath(arch);
```

Comment:
Is the `: string` actually necessary?
```suggestion
		const qemuBinPath = await qemuMod.getQemuPath(arch);
```

Reply:
Done